### ISSUE TITLE
When using the vite vue plugin to handle sfc src referencing scss files, you must explicitly indicate the lang.

### ISSUE THREAD

**Issue by skywalker512** [farm-fe/farm#967]
### Steps to reproduce

There is a Vue SFC file here that references an SCSS file, and this file has SCSS syntax.

```vue
<style scoped module src="./app.scss"></style>
```

There will be the following errors.

```
Farm ] Failed to start the server: Error: {"name":"CssSyntaxError","reason":"Unknown word","file":"/data/co/github/farm-vue-reproduce/apps/sfc-src-scss/src/App.vue","source":".logo {\n  height: 6em;\n  padding: 1.5em;\n  will-change: filter;\n  transition: filter 300ms;\n}\n.logo:hover {\n  filter: drop-shadow(0 0 2em #9f1a8faa);\n}\n.logo.vue:hover {\n  filter: drop-shadow(0 0 2em #42b883aa);\n  // Path: apps/sfc-src-scss/src/app.vue\n}\n","line":12,"column":6,"endLine":12,"endColumn":10,"input":{"column":6,"endColumn":10,"endLine":12,"line":12,"source":".logo {\n  height: 6em;\n  padding: 1.5em;\n  will-change: filter;\n  transition: filter 300ms;\n}\n.logo:hover {\n  filter: drop-shadow(0 0 2em #9f1a8faa);\n}\n.logo.vue:hover {\n  filter: drop-shadow(0 0 2em #42b883aa);\n  // Path: apps/sfc-src-scss/src/app.vue\n}\n","url":"file:///data/co/github/farm-vue-reproduce/apps/sfc-src-scss/src/App.vue","file":"/data/co/github/farm-vue-reproduce/apps/sfc-src-scss/src/App.vue"},"loc":{"file":"/data/co/github/farm-vue-reproduce/apps/sfc-src-scss/src/App.vue","line":29,"column":6},"code":"PLUGIN_ERROR","plugin":"vite:vue","id":"src/app.scss?vue&type=style&index=0&src=7a7a37b1&scoped=7a7a37b1&lang.module.css","hook":"transform"}
```

Must explicitly indicate lang.

```vue
<style scoped module src="./app.scss" lang="scss"></style>
```

https://github.com/skywalker512/farm-vue-reproduce/tree/main/apps/sfc-src-scss

### What is expected?

Work like Vite.

### What is actually happening?

vite will only retrieve lang from the attrs of the style tag to fill in lang.xxx, if not, it will default to using css.

https://github.com/vitejs/vite-plugin-vue/blob/plugin-vue%405.0.3/packages/plugin-vue/src/main.ts#L523-L546

Will generate such an id.

```
src/app.scss?vue&type=style&index=0&src=7a7a37b1&scoped=7a7a37b1&lang.module.css
```

However, in the farm, CSS is the first to be matched.

https://github.com/farm-fe/farm/blob/v0.16.5/packages/core/src/plugin/js/utils.ts#L74-L79

<img width="978" alt="image" src="https://github.com/farm-fe/farm/assets/12960906/7d892310-8b8f-429d-b7d2-c89ce271957d">

### System Info

```Shell
N/A
```

**Comment by wre232114** [comment]
Thanks for reporting issues. I'll make`scss` match first.

**Comment by wre232114** [comment]
fixed in @farmfe/core@0.16.7
